chromiumcontent/chromiumcontent.gypi

```diff
@@ -122,7 +122,7 @@
           },
         },
       }],
-      ['_target_name in ["gtk2ui", "devtools_http_handler", "devtools_discovery", "http_server"]', {
+      ['_target_name in ["gtk2ui", "devtools_http_handler", "devtools_discovery", "http_server", "desktop_capture", "webrtc_common", "system_wrapper", "rtc_base", "rtc_base_approved", "libyuv"]', {
```

Comment:
I think only `gtk2ui` should be listed here, it should be a mistake that `devtools_http_handler` and others were included.